### DISCUSSION TITLE
New Area for Discussions

### DISCUSSION THREAD

**Discussion by midimusic** [GrandOrgue/grandorgue#31] [General]
I have just logged in to this github discussion for the first time and am checking that it works with my login. It does. I assume this is now the normal place for GO discussions.

I have just installed Ubuntu Studio on my old 27" iMac in place of the Apple OS. I am using the whole disk and have got rid of all traces of the old OS. It now boots and runs Ubuntu. One of the problems is that the screen resolution is far to high for normal apps - they are much too small. I have tried the 2X screen magnification but it works rather badly and only on some apps. So I have gone back and told the system that the resolution is 1920X1080. This wastes the high resolution capability but does provide the proper sizes.

I tried to **install GO** but the installer would not install and said that there were dependencies which were not available without saying what they were- so _very unhelpful_. My previous ubuntu GO installation on a different system went OK so I am at a loss to know why GO will not install. Any suggestions?

csw900

**Comment by larspalo** [comment]
You could start with checking that the following packages exist on your system:
depends=('alsa-lib' 'jack' 'wxgtk3' 'libsystemd' 'fftw' 'wavpack' 'zlib')
Also, it could be helpful to know exactly what version (and from where) of GO you're trying to install.

**Reply by oleg68** [reply]
> One of the problems is that the screen resolution is far to high for normal apps - they are much too small. I have tried the 2X screen magnification but it works rather badly and only on some apps.

GrandOrgue allows to scale almost all screen pannels.

> I tried to install GO but the installer would not install and said that there were dependencies which were not available without saying what they were- so very unhelpful. My previous ubuntu GO installation on a different system went OK so I am at a loss to know why GO will not install. Any suggestions?

Sometimes reading [INSTALL.md](https://github.com/GrandOrgue/grandorgue/blob/master/INSTALL.md) may help you.
